libs/Matchmaking/test/Matchmaking.PeerDiscovery.Test/Matchmaking.PeerDiscovery.Test.csproj

```diff
@@ -4,6 +4,8 @@
     <TargetFramework>netcoreapp2.1</TargetFramework>
 
     <IsPackable>false</IsPackable>
+    <UseNugetPackages Condition="'$(UseNugetPackages)'==''">$(Configuration.EndsWith('nuget'))</UseNugetPackages>
+    <Configurations>Debug;Release;Debug-nuget;Release-nuget</Configurations>
```

Comment:
We can also define different combinations mixing&matching nuget/local if needed.